src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java

```diff
@@ -46,12 +46,16 @@ public FtpClient(
     }
     // endregion
 
-    public void upload(PdfDoc pdfDoc) {
+    public void upload(PdfDoc pdfDoc, boolean isSmokeTest) {
         Instant start = Instant.now();
 
         runWith(sftp -> {
             try {
-                String path = String.join("/", configProperties.getTargetFolder(), pdfDoc.filename);
+                String folder = isSmokeTest
+                    ? configProperties.getTargetFolder()
```

Comment:
These two lines should be swapped. Worth verifying which folder we write to in tests.